src/olympia/ratings/models.py

```diff
@@ -358,10 +359,29 @@ class RatingAggregate(ModelBase):
     count_5 = models.IntegerField(default=0, null=False)
 
 
+DOT = '.'
+ALPHANUMERIC_REGEX = r'[^\w]+|_+'
+
+
+def word_validator(value):
+    if DOT not in value and list(re.split(ALPHANUMERIC_REGEX, value)) != [value]:
+        raise ValidationError(
+            _('%(value)s contains a non-alphanumeric charactor.'),
+            params={'value': value},
+        )
+
+
 class DeniedRatingWord(ModelBase):
     """Denied words in a rating body."""
 
-    word = models.CharField(max_length=255, unique=True)
+    word = models.CharField(
+        max_length=255,
+        unique=True,
+        help_text='Can only contain alphanumeric charactors ("\\w", exc. "_"). '
+        'If contains a "." it will be interpretted as a doman name instead, '
+        'and can contain any charactor',
```

Comment:
```suggestion
        help_text='Can only contain alphanumeric characters ("\\w", exc. "_"). '
        'If contains a "." it will be interpreted as a domain name instead, '
        'and can contain any character',
```